content/assignments/hello-world/project-intro/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "1. Project Introduction"
+date: 2023-05-25T12:55:09-05:00
+draft: false
+weight: 1
+originalAuthor: John Woolbright # to be set by page creator
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Your first assignment is mostly an exercise in following instructions. We'll ask you to modify an existing program so that, when run, the program prints "Hello world!".
+
+{{% notice red Warning "rocket" %}}
+In order to complete this assignment there are a few things that you will need to have already completed! You can find these items below:
+
+1. [Installed Git]({{< relref "../../../installations/install-git/" >}})
+1. [Installed Visual Studio Code]({{< relref "../../../installations/install-visual-studio/" >}})
+1. [Installed Node]({{< relref "../../../installations/install-node/" >}})
+{{% /notice %}}
+
+Programming jobs often require you to get familiar with, or at least make use of, additional technologies. Here, you'll be using a few web applications, like GitHub and Repl.it, to write your code, run it, save it, and submit your assignment.
+
+So while the coding task may appear straightforward, you'll be introduced to a set of instructions, or **workflow**, that is important for the rest of your success in this class.
+
+## Requirements
+
+1. Sign up for a GitHub account.
+1. Change the code in Visual Studio Code.
+1. Commit the code.
+1. Push the code to your assignment repository.
+1. Make sure your assignment passes the test.
+
+### GitHub
+
+If you haven't done so already, create a [GitHub account](https://github.com/join?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F&source=header-home).
+
+### Assignment Invitation
+
+{{% notice blue Tip "rocket" %}}
+Before starting on any assignment, make sure you are signed into your GitHub account!
+{{% /notice %}}
+
+First, find `Assignment #0` in Canvas and click on the starter code repo link. Fork the repository to your personal profile, copy the repo’s URL for cloning, and clone the project to a directory on your machine.
+
```

Comment:
Suggestion add a section about enabling actions for the autograding.  Include brief description of how it works.

Reply:
There is a description of github actions in the test-and-commit chapter. I believe this should be adequate for the learners to work through the assignment and learn a bit more about github actions. I will add a link to the actions documentation on github for additional support.